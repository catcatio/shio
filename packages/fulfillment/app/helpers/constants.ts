import { join } from "path";
import yaml = require('js-yaml');
import fs = require('fs');

export function getFulfillmentDevelopmentConstant() {
  var doc = yaml.safeLoad(fs.readFileSync(join(__dirname, '../../env.example.yaml'), 'utf8'));
  return {
    datastoreNamespace: doc.env_variables.SHIO_FULFILLMENT_DATASTORE_NAMESPACE,
    projectId: doc.env_variables.SHIO_FULFILLMENT_PROJECT_ID,
    storageName: doc.env_variables.SHIO_FULFILLMENT_STORAGE_NAME,
  }
}