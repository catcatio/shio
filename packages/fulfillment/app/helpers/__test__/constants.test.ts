import { getFulfillmentDevelopmentConstant } from "../constants";

describe('constant test',() => {


  test('getFulfillmentDevelopmentConstant', () => {
    const constant = getFulfillmentDevelopmentConstant()
    expect(constant).toMatchSnapshot()
  })

})