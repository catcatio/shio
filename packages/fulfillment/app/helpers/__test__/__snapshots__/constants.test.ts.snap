// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constant test getFulfillmentDevelopmentConstant 1`] = `
Object {
  "datastoreNamespace": "fulfillment-development",
  "projectId": "catcat-development",
  "storageName": "catcat-development",
}
`;
